Config/Test_Subscriptions.storekit

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
was this auto generated?

Reply:
The StoreKit configuration file is created following the instructions in the StoreKit testing document, shared on Slack; so it's not auto-generated.